### PR TITLE
Concat

### PR DESCRIPTION
**Pull request by AmaranthineCodices**
Adds the concat operator (`..`) as a binary operation, with a test case and all that.

### CODE REVIEW

File: src/tokenizer.rs [hunk 1]
```diff
@@ -21,6 +21,7 @@ pub enum Symbol {
     Slash,
     Caret,
     Hash,
+    Concat,
```
Comment (LPGhatguy) [inline src/tokenizer.rs]: In the tokenizer, I've been trying to keep the names of the symbols separate from their actual semantics in Lua.

Is there a better name for `..` that doesn't describe what it does?

Comment (AmaranthineCodices) [reply src/tokenizer.rs]: Ergh, not sure. I noticed that, but couldn't think of another name. Possibly `TwoDots` or `TwoPeriods`?

Comment (LPGhatguy) [reply src/tokenizer.rs]: `DotDot`, vs `DotDotDot`? 😅 

`TwoDots` probably works.

Comment (AmaranthineCodices) [reply src/tokenizer.rs]: Alright; I've made the change!

**Comment by LPGhatguy** [comment]
That was super easy!

**Pull request merged**